helpers/typeConvert.js

```diff
@@ -1,6 +1,7 @@
 const toSafeName = require("./toClassName");
 
-const fromFormat = (propFormat, shouldBox) => {
+const fromFormat = (propFormat, shouldBox, inDeclaration) => {
+  let required_object_prefix = !shouldBox && inDeclaration ? "@NonNull " : "";
```

Comment:
```suggestion
  const notNullPrefix = !shouldBox && inDeclaration ? "@NonNull " : "";
```

Reply:
Not sure this has been resolved? e.g. the casing is inconsistent

Reply:
Ohh, sorry didn't realise the casing change! I'll fix that as well.

Reply:
Changed the var name and casing 👍